lib/ipmi_vita.c

```diff
@@ -368,20 +328,19 @@ ipmi_vita_get_vso_capabilities(struct ipmi_intf *intf)
 	}
 
 	printf("VSO Standard      : %s\n",
-		(rsp->data[3] & 0x3) == 0 ? "VITA 46.11" : "RESERVED");
+	       (rsp->data[3] & 0x3) == 0 ? "VITA 46.11" : "RESERVED");
```

Comment:
Like here. I wish this `0x3` was a macro as well.

Reply:
separate PR.